### PR TITLE
kraken

### PR DESCRIPTION
**Pull request by shadykube**

### CODE REVIEW

File: rootchpasswd/Dockerfile.rootchpasswd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+RUN echo 'root:Asdf_123' | chpasswd && \
```
Comment (Plazmaz) [inline rootchpasswd/Dockerfile.rootchpasswd:3]: This is interesting. Would you expect a secret tool to detect Asdf_1234 here? That'd be something I haven't seen before.

File: algolia/algolia-secret.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+def main():
+     x-algolia-api-key=a4d9d38893a8d71cd78572470a328dd4
```
Comment (Plazmaz) [inline algolia/algolia-secret.py:2]: This is not valid python code.

**Review by Plazmaz — CHANGES_REQUESTED** [review]
First, thanks for the PR! These look useful. I have a few requests if possible:

1. Could you please move these into folders that match the existing organization in this repo? (Categorized into web, db, cloud, and misc, outside of keys that need to be in a specific filepath to be recognized. You could definitely keep the folders so it'd be web/coinbase/coinbase_legacy for example)
2. Could you update .leaky-meta/secrets.csv to include these secrets?

File: algolia/algolia-secret.py (hunk 2 above)
Comment (hihihaha777) [reply algolia/algolia-secret.py:2]: lol

**Pull request closed without merging**